do/private/build_cmd

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -e
+
+pushd build > /dev/null 2>&1 || exit 1
+
+# now build with the intent to test
+cmake -DCMAKE_BUILD_TYPE=Release ..
+make -j$(nproc) package
+ls *.deb > .deb
+
+popd > /dev/null 2>&1
```

Comment:
If the code errors in lines 7, 8, or 9 fails, the final popd will not happen. Not sure it's a big deal.

Reply:
Actually, I'm not even using that file.  Deleted.